src/js/components/molecules/tabs/tabs.ts

```diff
@@ -29,23 +29,21 @@ export class Tabs extends Vue implements TabsProps {
     setActiveTab() {
         const tabs = this.$refs.tab;
 
-        if (!tabs) { return; }
+        if (!tabs) return;
 
         tabs[this.activeTab].scrollIntoView({ behavior: 'smooth', inline: 'center', block: 'nearest' });
 
         setTimeout(() => this.positionIndicator(), 50);
     }
 
-    getLinkClassList(index) {
-        return {
-            'tabs__list_link--active': index === this.activeTab,
-        };
-    }
+    getLinkClassList = index => ({
```

Comment:
wrap the parameter of an arrow function even when it's a single parameter. if the eslint rule it not set to that, update it.

also, get used to adding a return value type to functions.

```suggestion
    getLinkClassList = (index): boolean => ({
```

Reply:
I understand the typing, but I don't see why I should wrap the parameter in parentheses? Is that in our coding guidelines?

Also, shouldn't the return type be `object` instead of `boolean`? It returns an object, right?

Reply:
yes, that's the style we've been using for at least 6 months. and yes, i got the type wrong - good catch.

Reply:
I noticed that I can't do the typing without the parentheses, so that's why you wanted me to add them :) Fixed it